data/coalition missions.txt

```diff
@@ -18,6 +18,7 @@ mission "Coalition: First Contact"
 	on offer
 		"reputation: Coalition" += 10
 		"reputation: Heliarch" += 10
+		log "Made contact with the Coalition, an alliance of giant beetles, giant spiders and centaurs in the galactic south. They're ruled by 'Heliarchs', and they overthrew the Quarg. Seems they see them as oppressors."
```

Comment:
`'Heliarchs',` -> `'Heliarchs,'`
Comma after `giant spiders`, as it's part of the writing style of the game to have a comma before the and in a series of things. https://github.com/endless-sky/endless-sky/blob/0af9750fd2de030631c8c686d387d741ede2785c/data/free%20worlds%20intro.txt#L13
I think It'd change up the last two sentences. `They're ruled by a group known as 'Heliarchs.` They overthrew the Quarg millenia ago, who they seem to see as oppressors.`
Also, maybe change it so that Heliarchs is surrounded by double quotes and the entire log is surrounded in backticks, for consistancy (see Hai engines).